components/engine_traits/src/region_cache_engine.rs

```diff
@@ -119,6 +120,8 @@ pub struct CacheRegion {
     pub start: Vec<u8>,
     // data end key of the region range, equals to data_end_key(region.end_key).
     pub end: Vec<u8>,
+    // does not schedule a load if the region is in flashback
+    pub in_flash_back: bool,
```

Comment:
I feel not very well to put it in `CacheRegion` as it is only useful in `prepare_for_region` when pending region misses flashback event.  What about put in as a separate field in `prepare_for_region` instead? It can be misused as the `in_flash_back` in RegionMeta can not represent the real current status. 
I prefer to not put too many unrelated fields into this struct as it is used everywhere.